src/OnlineSales/Entities/EmailLog.cs

```diff
@@ -0,0 +1,55 @@
+﻿// <copyright file="EmailLog.cs" company="WavePoint Co. Ltd.">
+// Licensed under the MIT license. See LICENSE file in the samples root for full license information.
+// </copyright>
+
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Text.Json.Serialization;
+
+namespace OnlineSales.Entities
+{
+    public enum EmailStatus
+    {
+        NOTSENT = 0,
```

Comment:
@gsplakmal is there any kind of convention why you use UPPER CASE for enumerable? Can we use NotSent, Send instead?

Reply:
Updated all enums as suggested pattern